core/model.py

```diff
@@ -38,6 +38,12 @@ def _predict(self, x):
         else:
             output_json['distance'] = 0
 
-        output_json['candidates'] = [dimsim.get_candidates(i, mode, theta) for i in input_strings]
+        output_json['candidates'] = {}
+        output_json['candidates']['first_word'] = dimsim.get_candidates(input_strings[0], mode, theta)
+
+        if len(input_strings) == 2:
+            output_json['candidates']['second_word'] = dimsim.get_candidates(input_strings[0], mode, theta)
+        else:
+            output_json['candidates']['second_word'] = [""]
```

Comment:
Is it better to return an empty list here? Rather than a single-entry list with empty string?

Reply:
This was for flask because it looks for a List of String. I can try `[None]`.